src/Apps/Artist/Routes/Consign/Utils/getConsignmentData.ts

```diff
@@ -0,0 +1,79 @@
+/**
+ * Passed https://docs.google.com/spreadsheets/d/1EicHOCyCozsPBkVOkQGTTUtEds7qQuacUAgLy-C2-Nk/edit#gid=0
```

Comment:
Do we want this URL visible to the world?

Reply:
I thought about that too, it does have proper permissions so that it's not actually exposing any data.